config/routes.rb

```diff
@@ -639,4 +639,8 @@
   if OpenProject::Configuration.lookbook_enabled?
     mount Lookbook::Engine, at: "/lookbook"
   end
+
+  constraints(->(req) { User.exists?(id: req.session[:user_id], admin: true) }) do
```

Comment:
This needs to be configurable. On SaaS, you'd otherwise see all jobs in all schemas

Reply:
True. With current modification to SaaS the dashboard will have some bugs there. So, I left it enabled for dev only.